packages/scss/src/components/_textarea.scss

```diff
@@ -59,15 +59,15 @@
 		background-repeat: repeat-x;
 		background-size: 3px 1px;
 		box-shadow: none;
-		color: $greyLighter;
+		color: _get($luTheme, "palette.colors.greyLight");
 		cursor: not-allowed;
 	}
 }
 
 .textarea {
 	&.is-filled {
 		.textarea-label {
```

Comment:
Yep! 

With the methodology I follow each Sass components have to be separated. 
If some code is used in my places, we can use a mixin. 
But as it lead to less clarity I think it's okay to have a bit of duplicated code in 2 or 3 files in this case.